src/order/zorn.lean

```diff
@@ -92,6 +92,24 @@ lemma chain.mono {c c'} :
   c' ⊆ c → chain c → chain c' :=
 pairwise_on.mono
 
+lemma chain_univ [is_trichotomous α r] :
+  chain (univ : set α) :=
```

Comment:
Could `univ` be switched to an arbitrary set here?

Reply:
Hmm... yeah, actually.